cypress/support/step_definitions/validations.md

```diff
@@ -412,3 +445,156 @@ While validating, if a key is present in both fcs-validation jsons (eg: cypress/
             }
         ]
 }
+
+
+## Error Content Validation
+
+### Background
+
+Validating the error content obtained in the response. The validation is done against a source of truth, which contains an array of error codes and messages etc. The goal is to ensure that the error codes and messages in the response match the expected values defined in the source of truth.
+
+### Default Validation Types in FCS
+
+FCS having only 4 types for Validation error objects [errorContentObjects.json](../../fixtures/objects/errorContentObjects.json)
+
+- NOT_SUPPORTED
+- NOT_PERMITTED
+- NOT_AVAILABLE
+- INVALID_TYPE_PARAMS
+
+#### Format
+
+```
+{
+    "<Error object name>": {
+        "type": "schemaOnly"
+    }
+}
+```
+
+#### Params:
+
+| Param             | type   | Description                                                                                                                     |
+| ----------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| Error object name | string | Error object key name, which is given in the test case                                                                         |
+| type              | string | The value which indicates the type of validation.                                                                               |
+| schemaOnly        | string | By default FCS having validation as `schemaonly`, which will not perform content validation instead stops at schema validation. |
+
+#### Example:
+
+```
+    "NOT_SUPPORTED": {
+        "type": "schemaOnly"
+    }
+```
+
+Note:
+
+- Error content validation will be happened only when override objects are added from the config module.
+- If error type having `schemaOnly` as shown in example, exection will stop after schema validation and testcase will PASS.
```

Comment:
Replace below 
Validation objects can be added in config module: cypress/fixtures/objects/errorContentObjects.json file.
If an object in the config module shares the same key name as an error object defined in FCS, FCS error object will be overriden.

with 

-  User can add error code validations to error validations by overriding in config module in the cypress/fixtures/objects/errorContentObjects.json file.

- For that, they would need to use the same key name as any of the default error objects defined in FCS.

Format 
Below is the format need to be followed while adding override in config module

{
    "INVALID_TYPE_PARAMS": {
        "type": "errorValidationFunction",
        "validations": [
            {
                "type": {
                    "errorCode": [ERROR_CODE1, ERROR_CODE2]
                }
            }
        ]
    }
}

Note: FCS expects the error object to be defined in the above format. Any deviation from the above format would cause failures in error content validation.

Reply:
Replace **Any additional validations needed, that can be done using custom validation** with **Custom validation support**

Then add details as shown below
Users can further add their own error validations using custom validation objects. 

- To do custom validation, error object must have the type as custom shown in below format.
- The value given for assertionDef represents the function name and this function should be defined in config module: cypress/fixtures/customValidations/

Reply:
addressed the comment